### PR TITLE
Add ShiftStack 4.5 RN items

### PR DESCRIPTION
**Pull request by maxwelldb**
FYI @codyhoag @jeana-redhat 

Still getting feedback about what to put here.

### CODE REVIEW

**Comment by maxwelldb** [comment]
@racedo @EricArrakis We're starting to pull together 4.5 release notes. Let me know if what's up here seems anemic or suspect!

@luis5tb Which of the Kuryr items do you think should get an add here?

**Comment by luis5tb** [comment]
> @racedo @EricArrakis We're starting to pull together 4.5 release notes. Let me know if what's up here seems anemic or suspect!
> 
> @luis5tb Which of the Kuryr items do you think should get an add here?

I would mention the support for OSP upgrades to make use of new Octavia functionality, specifically mention the removal of the hack for DNS resolution over TCP.

Other than that we have other 2 more or less internal things, not sure if work highlighting:
- Add support for must-gather, grabbing a lot of information about things related to kuryr
- Added a new CRD named KuryrNetworks to keep track of the OpenStack network/subnet associated to the OCP namespace

**Comment by EricArrakis** [comment]
We also added Anti-Affinity and Manila CSI

Is "Additional networks for clusters" the "Secondary NIC" support?

cc @iamemilio @Fedosin @pierreprinetti @mandre

**Comment by maxwelldb** [comment]
Yes. "Additional networks" is the phrase used upstream, so I just used that.

**Comment by pierreprinetti** [comment]
>Anti-affinity wasn't a docs required change, so I'm not too sure about what to say there. Any help is appreciated

Here's how I'd sum up the change:
_The Compute nodes are now scheduled on separate physical hosts if those are available._

**Comment by maxwelldb** [comment]
Thoughts on https://github.com/openshift/openshift-docs/pull/23309/commits/e24bdc793656864a075f927358b47d48443aa0f9, @luis5tb?

**Comment by maxwelldb** [comment]
@luis5tb How about 45c8539, now?

File: release_notes/ocp-4-5-release-notes.adoc [hunk 1]
```diff
@@ -101,6 +101,27 @@ type of private DNS zone.
...
+
+{product-title} {product-version} introduces support for multiple networks in clusters that run on {rh-openstack}.
+You can specify these networks for both control plane and compute machines during installation.
+
+[id="ocp-4-5-openstack-kuryr-octavia"]
+==== Improved {rh-openstack} load balancer upgrade experience for clusters that use Kuryr
+
+Previously, clusters that were installed on {rh-openstack} version 13 and then upgraded to {rh-openstack} version 16 did not take full advantage of {rh-openstack}'s load balancing features.
+
+Upgraded clusters that use Kuryr now have improved support for Octavia load balancing services, including the Octavia OVN provider driver.
```
Comment (luis5tb) [inline release_notes/ocp-4-5-release-notes.adoc:121]: This should be:
Clustter that use Kuryr now have improved support for Octavia load balancing services on {rh-openstack} clusters that were updated from 13 to 16, including the Octavia OVN provider driver

**Comment by maxwelldb** [comment]
@codyhoag @jeana-redhat I think this is ready for peer review.

**Comment by jeana-redhat** [comment]
LGTM but I defer to @codyhoag

**Comment by maxwelldb** [comment]
@codyhoag Let me know if you have any concerns about this.

**Review by codyhoag — COMMENTED** [review]
Sorry for the delay; one general question: 

These RHOSP entries are all listed under the Machine API section. Should they be moved to the installation/upgrade section?

Otherwise, LGTM!

**Comment by maxwelldb** [comment]
@codyhoag :scream: Don't know how that happened. Yeah, installation. Thanks for the check. :+1:

**Comment by maxwelldb** [comment]
@codyhoag Oh, I'm seeing everything that should be under Installation there already. 

There's a single Node item to do with anti-affinity node scheduling. That seemed to make sense in that section to me. 

Gonna rebase presently, but could you confirm that we're seeing the same thing?

**Comment by maxwelldb** [comment]
Ah. Saw it once I fetched and rebased. Should be fixed in the next commit.

**Review by codyhoag — APPROVED** [review]
That section placement LGTM!

**Comment by maxwelldb** [comment]
@codyhoag Thanks!

**Pull request merged**